pos_order_mgmt/__manifest__.py

```diff
@@ -9,6 +10,7 @@
     'category': 'Point of Sale',
     'author': 'GRAP, '
               'Tecnativa, '
+              'Domatix, '
```

Comment:
This contribution doesn't grant you as co-author. And are you modifying something of Pierrick Brun's code. If not, you shouldn't put even your copyright. Adding in contributors is fine.